configs/setups/awiesm/awiesm.yaml

```diff
@@ -330,7 +340,17 @@ fesom:
         # ----------------------------------------------------
         choose_general.version:
                 "2.6":
-                        version: "2.6-paleodyn"
+                        version: "2.6"
+                        ALE_scheme: 'zstar'
+                        namelist_dir: ${fesom.model_dir}/config/
+                        add_namelist_changes:
+                                namelist.tra:
+                                        tracer_phys:
+                                                surf_relax_s: 0.0
+                        outdata_targets:
+                                fesom: "*.fesom.*.nc"
```

Comment:
Is this something version specific? I would like to use that for all versions of fesom2. Users that have sub-yearly runs are rare. IMO those users can then add the alternative output name scheme into the runscript.

Reply:
Otherwise this looks good.